neo-codap/src/graph/graph.tsx

```diff
@@ -323,6 +664,33 @@ export class GraphComponent extends React.Component<IGraphProps, IGraphState> {
                 .append('g')
                 .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
 
+        uniqueLegendValues.forEach((uniqueLegendValue, index) => {
+            const values: IGraphCoordinate[] = [];
+            const legendRange = legendAttrType && isNumeric ? legendAttrType.quantiles[index] : undefined;
+            xValues.forEach((iX: number, i: number) => {
+                let addPoint = false;
+                if (legendRange) {
+                    const legendValue = legendValues[i];
+                    addPoint = (legendValue !== undefined) &&
+                               (legendValue >= legendRange.min ) &&
+                               (legendValue <= legendRange.max);
+                }
+                else {
+                    addPoint = uniqueLegendValue === legendValues[i];
+                }
+                if (addPoint) {
+                    values.push({x: x(iX), y: y(yValues[i])});
+                }
+            });
+            coordinates[index] = values;
+        });
+
+        svg.append('rect')
+            .attr('class', 'background')
+            .attr('width', '100%')
+            .attr('height', '100%')
+            .on('click', () => this.handleBackgroundClicked());
```

Comment:
Couldn't this be simplified to
```
.on('click', this.handleBackgroundClicked)
```
since handleBackgroundClicked() is already defined as a double-arrow function?

Reply:
I was seeing this bound to the svg element in the debugger when I had it bound directly.  I'll double check if this was just the debugger being confused.